src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/ComInterfaceInfo.cs

```diff
@@ -74,7 +74,7 @@ public static DiagnosticOrInterfaceInfo From(INamedTypeSymbol symbol, InterfaceD
                     baseSymbol?.ToDisplayString(),
                     syntax,
                     new ContainingSyntaxContext(syntax),
-                    new ContainingSyntax(syntax.Modifiers, syntax.Kind(), syntax.Identifier, syntax.TypeParameterList),
+                    new ContainingSyntax(syntax.Modifiers.StripTriviaFromTokens(), syntax.Kind(), syntax.Identifier.WithoutTrivia(), syntax.TypeParameterList),
```

Comment:
Would it make more sense to perform the trivia clean-up in the `ContainingSyntax` class? This style of statement makes debugging very difficult, there are so many nested function calls (implicit and explicit) in the statement.

Reply:
It does seem like in almost all uses of `ContainingSyntax` we remove trivia.